src/app/guide/_components/GuideDetail.tsx

```diff
@@ -0,0 +1,54 @@
+import Image from 'next/image';
+import SVGTriangle from '@/styles/icons/triangle.svg';
+
+interface ImageWithDescriptionProps {
+  description: string;
+}
+function ImageWithDescription({ description }: ImageWithDescriptionProps) {
+  return (
+    <div className="flex w-full flex-col items-center">
+      <div className="h-12 w-full bg-white" style={{ paddingTop: '100%' }} />
+      <div className="font-primary-darkblue flex flex-row items-center gap-1 pt-2 font-sfpro text-xs font-extrabold">
+        <SVGTriangle />
+        {description}
+      </div>
+    </div>
+  );
+}
+
+interface GuideDetailProps {
+  title: string;
+  children: React.ReactNode;
+  description1: string;
+  description2: string;
+  description3: string;
+  description4: string;
+}
```

Comment:
```suggestion
interface GuideDetailProps {
  title: string;
  children: React.ReactNode;
  examples: {src: string; description: string;}[];
}
```

후에 사진 넣을 부분을 고려해서 이런형태로 타입을 만들면 더 깔끔할 것 같습니다! 이런 방법을 사용한다면 아래에 있는 JSX 코드는 다음과 같이 작성하면 더 확장성 있을 것 같아요!

```tsx
   {examples.map(example => <ImageWithDescription description={example.description} />)}
```

Reply:
이 부분이 잘 이해가 안되는데, 위 타입선언이 아래 코드로 확장할수있다는 것에대해서 좀 더 자세히 설명해주실수있나요??
@

Reply:
원래 이미지가 들어가는 사진인데 지금은 제가 임의로 도형으로 넣어놓은 상태거든요!
이미지 넣어서 일상님 코드 토대로 고쳤는데 코드로 보시면 이해가 빠르실 것 같아요 (만약에 맞다면 ㅎㅎ..)  빨리 수정해서 다시 올려두겠습니다!